spec/models/kit_order_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+describe KitOrder do
+  let(:blank_values) { [nil,''] }
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.